source/includes/note-collections-in-mixed-sync.rst

```diff
@@ -0,0 +1,23 @@
+.. note:: Collections in Mixed fields supported in Apps created after March XX, 2024
+    
+    App Services Apps created *after* March XX, 2024 can store collections
```

Comment:
It would be nice to involve some form of the word document / json in here since that is what this is doing and is what people ask about (IE, why can't I just store a document)

Reply:
I'll update to better incorporate that wording (and possibly in some other places, if it makes sense)